code/index.html

```diff
@@ -1,37 +1,87 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="stylesheet" href="style.css" />
-    <title>Technigo Project Cluedo</title>
-  </head>
-  <body>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css" />
+  <title>🔍 Sandrine's Cluedo</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link href="https://fonts.googleapis.com/css?family=Nosifer" rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <h1>Cluedo mystery picker</h1>
+  </header>
+  <main>
     <section class="cards">
       <article class="killer-deck" id="killerCard">
-        <p>KILLER</p>
+        <p class="card-headers">KILLER</p>
+        <div class="loader" id="killerLoader">
+          <i class="fa fa-circle-o-notch fa-spin"></i>
```

Comment:
Great that you used an icon tag for the loader!

Reply:
Kind of a quick fix but I can recommend it, it is very easy to use. I first used padding to place it in the middle of the card and it spun in a very comical way 😂, margin works better.